cypress/support/commands.js

```diff
@@ -150,6 +150,17 @@ Cypress.Commands.add('addAttachment', (cardName, attachmentLink, linkName) => {
     cy.get('[class="js-add-attachment-url"]').click();
 });
 
+Cypress.Commands.add('copyCard', (cardName, copyCardName, boardName, listName, cardPosition) => { 
+    cy.get('[class="list-card-title js-card-name"]').contains(cardName).click();
+    cy.get('[class="js-sidebar-action-text"]').contains('Copy').click();
+    cy.get('[class="js-autofocus"]').type(copyCardName);
+    cy.get('select.js-select-board').select(boardName+' (current)');
+    cy.get('select.js-select-list').select(listName);
+    cy.get('select.js-select-position').select(cardPosition);
+    cy.get('[class="nch-button nch-button--primary wide js-submit"]').click({force: true});
```

Comment:
This kind of selectors are not good, try to use only the minimum classes, not all of them, or find something else that can be used to get the element.